### ISSUE TITLE
rabbitmq plugin is missing flag -vault-active

### ISSUE THREAD

**Issue by dlinsley** [enaml-ops/omg-product-bundle#10]
Was expecting the `-vault-active` flag in all service plugins.  It appears to be missing with the rabbitmq plugin.  @zmb3 

`Incorrect Usage: flag provided but not defined: -vault-active`

**Comment by zmb3** [comment]
Yeah determined that vault active is not really necessary. If you specify a
vault domain and token that's enough to activate it.
